com.unity.UnityHub.appdata.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">com.unity.UnityHub.desktop</id>
+  <name>Unity Hub</name>
+  <developer_name>Unity Technologies</developer_name>
+  <summary>Real-time game engine</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://unity3d.com/</url>
+  <icon type="remote" height="256" width="256">https://raw.githubusercontent.com/kirbyfan64/flathub/unity/Unity.png</icon>
+  <description>
+    <p>Unity is used to create half of the world’s games. Our real time platform, powered by tools and services, offer incredible possibilities for game developers, and creators across industries and applications.</p>
```

Comment:
We should add some disclaimer that this Flatpak wrapper is not supported by or verified by Unity Tech - eg https://flathub.org/apps/details/com.slack.Slack